eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,9 +68,29 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+        displayName: "Install dependencies"
+
+      - script: |
+          node common/scripts/install-run-rush.js lint $(rushFlags)
+        displayName: "Lint libraries"
+
+      - task: CopyFiles@2
+        inputs:
+          contents: "sdk/**/**/*lintReport.html"
+          targetFolder: $(Build.ArtifactStagingDirectory)
+          flattenFolders: true
+        displayName: "Copy lint reports"
+
+      - task: PublishPipelineArtifact@1
+        condition: succeededOrFailed()
+        displayName: "Publish lint reports"
+        inputs:
```

Comment:
I think you need to specify the following as well:

```
    path: $(Build.ArtifactStagingDirectory)
```

Reply:
added `targetPath: $(Build.ArtifactStagingDirectory)`

Reply:
That works too, the docs are out of date - `path` is the official argument name but it’s aliased to `targetPath` for backwards compatibility.

Reply:
@mikeharder

Reply:
I opened a doc issue for this: https://github.com/MicrosoftDocs/vsts-docs/issues/4908

Reply:
Thank you Mike :)